Oma error optimization (#9025)
newrelic/docs-website: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx

At the next triaging meeting you can revisit these error groups to see if they can now be marked as resolved. As time goes by, you should start to see fewer new error groups appearing, and positive movement in your KPIs.

### Link errors to JIRA [#jira]
As you get into more edge-case or complex errors, you might find you need to ask for help from other teams. Linking errors inbox to Jira may help with this. [Connect your Errors inbox to Jira](/docs/errors-inbox/error-external-services#jira) to enable you to easily create tickets connected to error groups. You can control the information sent to Jira via Jira templates.

### Link errors to Slack [#slack]
As the velocity of errors coming into errors inbox goes down, a regular team session may no longer be a good use of time. An alternative is to [link Errors inbox to Slack](/docs/errors-inbox/error-external-services#slack) and either a) designate a rotating person to monitor the channel and resolve/ignore/assign error groups as they come in or b) allow the team to proactively respond to error groups.

### Use CodeStream [#codestream]

> Many of your error groups are going to require code changes to resolve. [Integrate CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to allow you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 

Reviewer: ```suggestion
Many of your error groups are going to require code changes to resolve. [Use CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
```

Reviewer 2: What do you think about: "Connect CodeStream to your New Relic account so you can open the offending..."

Also, "offending" is so funny :D

> The CodeStream integration gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.

Reviewer: ```suggestion
New Relic with CodeStream gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
```

Reviewer 2: I think replace "The CodeStream integration" with "New Relic with CodeStream" or even just "CodeStream" is the right call.
